frb_example/pure_dart/rust/c_output_path_bak/c_output.h

```diff
@@ -0,0 +1,1149 @@
+#include <stdbool.h>
```

Comment:
uber-nit: what is "c_output_path_bak"? e.g. rename to "c_output_path_extra"?

Reply:
Fixed in https://github.com/fzyzcjy/flutter_rust_bridge/pull/1024/commits/cbd8da41d3f47156452430b5a2774aa471e2ddb2.